docs/distributed/DESIGN.md

```diff
@@ -33,38 +31,19 @@ Distributed erasure coded configuration with 64 sets with 16 disks each.
 minio server http://host{1...16}/export{1...64}
 ```
 
-## Other usages
+## Architecture
 
-### Advanced use cases with multiple ellipses
+Expansion of ellipses and choice of erasure packs based on this expansion is an automated process in MinIO. Here are some of the details of our underlying erasure coding behavior.
 
-Standalone erasure coded configuration with 4 sets with 16 disks each, which spawns disks across controllers.
-```
-minio server /mnt/controller{1...4}/data{1...16}
-```
+- Erasure coding used by MinIO is [Reed-Solomon](https://github.com/klauspost/reedsolomon) erasure coding scheme, which has a total shard maximum of 256 i.e 128 data and 128 parity. MinIO design goes beyond this limitation by doing some practical architecture choices.
 
-Standalone erasure coded configuration with 16 sets 16 disks per set, across mnts, across controllers.
-```
-minio server /mnt{1..4}/controller{1...4}/data{1...16}
-```
+- Erasure pack is single most maximum erasure unit beyond which an object is not sharded. This is automatically calculated based on the number of disks on command line. MinIO supports unlimited number of disks but each erasure pack can be upto 16 disks and a minimum of 4 disks. Anything beyond 16 we saw that erasure coding can become chattier and do not have any performance advantages. Anything beyond 16 can also be considered as superfluous since 16 erasure pack itself can give you tolerance of 8 disks per object by default which is plenty in any practical scenario.
```

Comment:
```suggestion
- Erasure pack is a single erasure coding unit within a MinIO deployment. An object is sharded within an erasure pack. Erasure pack size is automatically calculated based on the number of disks. MinIO supports unlimited number of disks but each erasure pack can be upto 16 disks and a minimum of 4 disks. 

- We limited the number of drives to 16 for erasure pack because, erasure code shards more than 16 can become chatty and do not have any performance advantages. Additionally since 16 drive erasure pack gives you tolerance of 8 disks per object by default which is plenty in any practical scenario.
```